scala/private/phases/phase_dependency.bzl

```diff
@@ -85,11 +85,11 @@ def _get_unused_deps_mode(ctx):
 
 def _is_target_included(target, includes, excludes):
     for exclude in excludes:
-        if target.startswith(exclude):
+        if target.startswith(exclude) or target.startswith("@" + exclude):
```

Comment:
https://bazel.build/external/overview#canonical-repo-name
for blzmod labels can be like "@@..."